recipes/mss/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.2.0" %}
+
+package:
+    name: mss
+    version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        # mss import pyproj from basemap and that does not exist in 1.0.8
+        - basemap 1.0.7
```

Comment:
already in the repo
https://bitbucket.org/wxmetvis/mss/commits/37bdb9202aa521da1322f5f69d557802f7383710

but currently we would also need > 1.0.7 basemap-hires..  which is not automaticly recognized by basemap itselfs (I understand the reason). So somehow that dependency has to come in without making probs if its not there.